lms/djangoapps/discussion_api/api.py

```diff
@@ -36,7 +41,18 @@ def _get_course_or_404(course_key, user):
     return course
 
 
-def get_course_topics(course_key, user):
+def get_thread_list_url(request, course_key, topic_id_list):
+    """
+    Returns the URL for the thread_list_url field, given a list of topic_ids
+    """
+    path = reverse("thread-list")
+    query_list = [("course_id", course_key)]
+    for topic_id in topic_id_list:
+        query_list.append(("topic_id", topic_id))
```

Comment:
Rather than appending, do this:

```
query_list = [("course_id", unicode(course_key)] + [("topic_id", topic_id) for topic_id in topic_id_list]
```